src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:39919/organizations/org2/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:39919/project/issues?id=sample&pullRequest=5637&issues=AWcv1tyc6wY2FBB4XTZO&open=AWcv1tyc6wY2FBB4XTZO)

[ISSUE_KEY]: AWcv1tyc6wY2FBB4XTZO